bleep-core/src/scala/bleep/packaging/PackagedLibrary.scala

```diff
@@ -0,0 +1,6 @@
+package bleep
+package packaging
+
+import coursier.core.Dependency
+
+case class PackagedLibrary(asDependency: Dependency, files: Layout[RelPath, Array[Byte]])
```

Comment:
Array's tend to be a pain in the neck. Would [`ArraySeq`](https://www.scala-lang.org/api/2.13.x/scala/collection/immutable/ArraySeq.html) also work here?

Reply:
I sympathize, but not in this case. `Array[Byte]` is an optimal type for interop between different code paths (read from disk, extract from string, read from network) and then flushing to disk through `Files.write` or similar. 

I suppose we could hide it behind a `private[bleep]` modifier inside some wrapper class, but if you're the type to mutate random byte arrays coming through your code I don't suppose a `private` modifier will stop you :D

Reply:
`ArraySeq` is such a wrapper around `Array`, easy to turn into and from, back and forth, conveniently included in the standard library.
But hey, this is not a big deal.

Reply:
If one would want a immutable bytearray you can use ByteVector from scodec-bits,
In this case this probably is not really that interesting.